packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -150,82 +219,127 @@ class MemoryBodyState extends State<MemoryBody> {
 */
   }
 
+  static const minIncludeTextLeftButtons = 1300;
+  static const minIncludeTextRightButtons = 1100;
+
   Widget _leftsideButtons() {
-    return Row(
-      crossAxisAlignment: CrossAxisAlignment.center,
-      children: [
-        OutlineButton(
-          key: MemoryScreen.pauseButtonKey,
-          onPressed: controller.paused ? null : _pauseLiveTimeline,
-          child: const MaterialIconLabel(
-            Icons.pause,
-            'Pause',
-            minIncludeTextWidth: 900,
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: 6),
+      // Add a semi-transparent background to the
+      // expand and collapse buttons so they don't interfere
+      // too badly with the tree content when the tree
+      // is narrow.
+      color: Theme.of(context).scaffoldBackgroundColor.withAlpha(200),
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          OutlineButton(
+            key: MemoryScreen.pauseButtonKey,
+            onPressed: controller.paused ? null : _pauseLiveTimeline,
+            child: Label(
+              pauseIcon,
+              'Pause',
+              minIncludeTextWidth: 1300,
```

Comment:
use static const `minIncludeTextLeftButtons` from above. here and elsewhere

Reply:
Done.